### PR TITLE
add images to README.md

### PR DESCRIPTION
**Pull request by dmitryagapov**

### CODE REVIEW

**Comment by snosov1** [comment]
Adding images is great, the only thing is - I don't quite like having 4 images at the very top. Maybe, we can leave only one at the top and move other images to the bottom or other files?

File: web/README.md [hunk 1]
```diff
@@ -22,3 +24,13 @@ To start working with WEB OTE it is required to set up it locally. To do it choo
...
 
+# UI Examples
+Face Detection Page:
+![face-detection](.images/face-detection.png)
+
+Fine Tune Dialog:
+![fine-tube](.images/fine-tune.png)
+
+Assets Page:
+![assets](.images/assets.png)
```
Comment (alalek) [inline web/README.md:35]: BTW, Images resolution is 1848x1006.
Consider adapting its size for GitHub's README.md layout:
- hard to see details on FHD monitors
- avoid resize artifacts.

"Live" demo: https://github.com/openvinotoolkit/training_extensions/tree/add-images-to-readme/web

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
run tests

**Pull request merged**